lib/job-simulation.rb

```diff
@@ -5,11 +5,34 @@ class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    @jobs_available = jobs_available
+    @job_seekers = job_seekers
+    @workers = Stack.new
+    @waiting = Queue.new
+    n = 0
+    @jobs_available.times do
+      @workers.push("Worker ##{n+1}")
+      n += 1
+    end
```

Comment:
This works, but you also could do this, which reads easier but does the same thing:
```
jobs_available.times do |i|
  @workers.push("Worker ##{i+1}")
end
```